docs/data/tree-view/simple-tree-view/expansion/expansion.md

```diff
@@ -32,8 +32,33 @@ Use the `onItemExpansionToggle` prop to trigger an action upon an item being exp
 
 {{"demo": "TrackItemExpansionToggle.js"}}
 
-## Change item expansion
+## Imperative API
 
-You can use the `setItemExpansion` API method to imperatively change the expansion of an item:
+:::success
+To use the `apiRef` object, you need to initialize it using the `useTreeViewApiRef` hook as follows:
 
-{{"demo": "ChangeItemExpansion.js"}}
+```tsx
+const apiRef = useTreeViewApiRef();
+
+return <SimpleTreeView apiRef={apiRef}>{children}</SimpleTreeView>;
+```
+
+`apiRef` will be undefined during the first render and will then contain methods allowing you to imperatively interact with the Tree View.
```

Comment:
```suggestion
`apiRef` will be `undefined` during the first render. When initialised it will contain methods allowing you to imperatively interact with the Tree View.
```